### DISCUSSION TITLE
Send a message to specific player

### DISCUSSION THREAD

**Discussion by jonit-dev** [geckosio/geckos.io#163] [Help / Questions]
Hey folks, 

how do I send a message to a specific player?

In socket.io there's a .to('roomId') function.

What about here?

Thanks

**Comment by caramboleyo** [comment]
With `channel.emit`. You have to save the `channel.id` in `onConnection` and identifiy the user within that channel. A channel is always one connection between server and peer. Its not a channel like in television, its a channel like a river where you ship things through.

EDIT: or did you mean directly from peer to peer? Cause thats an intresting one, i don't think thats covered.
